css/style.css

```diff
@@ -1,19 +1,217 @@
+    list-style: none;
+    color: var(--grey-dark);
+}
+
+.nav__list__item {
+    margin: 0 1rem;
+    cursor: pointer;
+}
+
+.nav__list__item:hover {
+    color: var(--orange-dark);
+}
+
+.nav__list__item:focus {
+    font-weight: bold;
+    color: initial;
+}
+
+/* Main section */
+.main {
+    display: flex;
+    flex-direction: column;
+}
+
+/* Buttons */
+.btn--primary {
+    padding: 0.8rem 2rem;
+    border: 1px solid var(--orange-dark);
+    border-radius: 0.5rem;
+    font-size: 1rem;
+    background-color: var(--orange-dark);
+    color: var(--orange-light);
+}
+
+.btn--primary:hover {
+    color: var(--grey-dark);
+    background-color: var(--orange-light);
+}
+
+
+/* Hero section */
+.hero {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+    background-color: var(--grey-light);
+    background-image: url("../img/first-background.jpg");
+}
+
+.hero__title, .hero__subtitle {
+    color: var(--white);
+}
+
+.hero__title {
+    margin-bottom: 1.5rem;
+    font-size: 3.5rem;
+    font-weight: 300;
+}
+
+.hero__subtitle {
+    margin-bottom: 4rem;
+    font-size: 2rem;
+    font-weight: 100;
+}
+
+/* Description section */
+
+.description {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 80vh;
+    background-color: var(--white);
+}
+
+.description__title {
+    margin: 6rem 3rem 3rem;
+    font-size: 3rem;
+    font-weight: 300;
+}
+
+.description_container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
+
+.description__article {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem 2rem;
+    padding: 0 2rem;
+}
+
+.description__article__image {
+    height: 200px;
+    width: auto;
+}
+
+.description__article__caption {
+    padding: 1.5rem 0;
+    font-size: 1.5rem;
+}
+
+/* Subscription section*/
 
+.subscription {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: fit-content;
+}
+
+.subscription__quotation {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--orange-light);
+    width: 100%;
+    align-self: stretch;
+    text-align: center;
+}
+
+.subscription__quotation__text {
+    display: block;
+    width: 50%;
+    margin: 2rem;
+    font-size: 2rem;
+    font-style: italic;
+    font-weight: 300;
+}
+
+.text__span {
+    font-size: 3rem;
+    font-weight: 500;
+    color: var(--orange-dark);
+}
+
+/* Footer */
+
+.footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-top: 1px solid var(--grey-light);
+    margin: 4rem 2rem 0;
+
+}
+
+.footer__text {
+    padding: 2rem;
+    font-size: 1.5rem;
+}
+
+.footer__links__icon {
+    width: 30px;
+    height: auto;
+    margin: 0 1rem;
+    padding: 1rem;
+    border: 1px solid var(--grey-light);
+    border-radius: 50%;
+}
```

Comment:
Thats good Saadia, I had the problem with icons and i checked yours .css file, This is my opinion, you will be good developer, please go ahead.